src/_locales/hu/messages.json

```diff
@@ -55,72 +55,72 @@
         "message": "Firefox for iOS"
```

Comment:
In Hungarian is used "Firefox for iOS"?

Reply:
Yes, we treat it as a brand (proper noun).